packages/protocol/contracts/core/balanceSheet/BalanceSheetV1.sol

```diff
@@ -236,6 +236,41 @@ contract BalanceSheetV1 is
         }
     }
 
+    /// @inheritdoc IBalanceSheetV1
+    function getRepayBondAmount(
+        IHToken bond,
+        uint256 liquidationAmount,
+        IErc20 collateral
+    ) public view override returns (uint256 repayBondAmount) {
+        // Normalize the collateral amount.
+        uint256 normalizedLiquidationAmount;
+        uint256 collateralPrecisionScalar = 10**(18 - collateral.decimals());
```

Comment:
This should be enclosed within an `unchecked` block to save gas. The number of decimals is [guaranteed](https://github.com/hifi-finance/hifi/blob/0eeff4c32b51d9f4ba6e26b369e04561297dbf4b/packages/protocol/contracts/core/fintroller/FintrollerV1.sol#L153-L155) to be 18 or lower.

Reply:
You mean `collateralPrecisionScalar`?
```solidity
uint256 collateralPrecisionScalar;
unchecked {
    collateralPrecisionScalar = 10**(18 - collateral.decimals());
}
```

If so, we need to do the same for `getSeizableCollateralAmount(...)`.

Reply:
Yes I mean `collateralPrecisionScalar`. And yes you're right, we need to do the same in `getSeizableCollateralAmount`. But let's not make that change in this PR.